### DISCUSSION TITLE
APM beta v0.0.5

### DISCUSSION THREAD

**Discussion by marchbold** [airsdk/Adobe-Runtime-Support#1140] [Show and tell]
We just released the latest beta of apm (v0.0.5). The AIR Package Manager is at a functional level now and I'm really keen to get people involved! If you are interested please reach out and I'll help get you setup.

https://github.com/airsdk/apm

Key recent additions:
- Project setup and initialisation;
- Package installation and management;
- Ability to "check and install updates" for apm directly;
- Application descriptor generation;
- There are over 50 packages in the repository and we should have the majority of distriqt's in there within the next month;
- Harman will be pushing their PlayAssetDelivery ANE there shortly;

My favourite feature is the app descriptor generation. We have complete manifest additions / info additions / entitlements merging working so you can generate and update all the key parts for your application descriptor.

I'm planning on doing some walkthroughs over the next few weeks of normal use cases which hopefully will be a good starting point.

**Comment by ajwfrost** [comment]
Sounds awesome, thanks @marchbold!

App descriptor generation is great, particularly the ability to edit/update parts of this via the tool. I'll update my copy here and have a play :-)
